frontend/src/components/Maps.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react'
+import { Map, GoogleApiWrapper, Marker } from 'google-maps-react'
+import SideCard from './SideCard';
+const config = require("../config/constants.json")
+
+
+export class Maps extends Component {
```

Comment:
Why do you need to use a class component? What is the difference between a class component and a function component?

Reply:
React, for the development of the test I need to use the state and it is easy and fast to use with class components, and I know that it is better to use functional components when it is not necessary to use the state or the needs to access a life cycle hook

Reply:
Yes, that was the way a couple of versions ago, now they recommend using a hook called useState in case that you need it